libsolidity/ast/Types.h

```diff
@@ -1108,6 +1108,8 @@ class UserDefinedValueType: public Type
 	bool leftAligned() const override { return underlyingType().leftAligned(); }
 	bool canBeStored() const override { return underlyingType().canBeStored(); }
 	u256 storageSize() const override { return underlyingType().storageSize(); }
+	unsigned storageBytes() const override { return underlyingType().storageBytes(); }
+
```

Comment:
Can you also add `containsNestedMapping` just in case we extend this in the future?

Reply:
Also `hasSimpleZeroValueInMemory`

Reply:
And maybe `dataStoredIn`, but that's it :)